core/frontend/src/IModelApp.ts

```diff
@@ -320,8 +319,8 @@ export class IModelApp {
     this._applicationVersion = (opts.applicationVersion !== undefined) ? opts.applicationVersion : "1.0.0";
     this.authorizationClient = opts.authorizationClient;
 
-    this._hubAccess = IModelHubFrontend;
-    IModelHubFrontend.setIModelClient(opts.imodelClient);
+    if (undefined !== opts.hubAccess)

```

Comment:
this test is not necessary. There is no default, right?

Reply:
Removed, the HubAccess is now optionally defined and will throw in the openRemote method if it is undefined. In all other cases though, it is still valid to not be defined (i.e. Standalone, Snapshot, etc.).